demos/evp/aesgcm.c

```diff
@@ -69,20 +70,26 @@ void aes_gcm_encrypt(void)
     /* Output encrypted block */
     printf("Ciphertext:\n");
     BIO_dump_fp(stdout, outbuf, outlen);
+    /* Compare ciphertext against expected */
+    verifyCt = memcmp(gcm_ct, outbuf, sizeof(gcm_ct));
+    printf("Ciphertext Verify %s\n", verifyCt == 0 ? "Successful!" : "Failed!");
```

Comment:
I'm not convinced this is necessary in a demo. This isn't a test, its supposed to demonstrate how best to do things. You'd never do this in any real code so why do it here? I think it could just add confusion.